packages/langchain/README.md

```diff
@@ -0,0 +1,122 @@
+# @sap-ai-sdk/langchain
+
+This package contains Langchain compliant models, based on the @sap-ai-sdk clients.
+
+### Installation
+
+```
+$ npm install @sap-ai-sdk/langchain
+
+$ npm install @langchain/openai // if you want to use OpenAI models
+```
+
+## Pre-requisites
+
+- [Enable the AI Core service in BTP](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/initial-setup).
+- Project configured with Node.js v20 or higher and native ESM support enabled.
```

Comment:
[req]All other points are in imperative. I don't care whether you use the imperative or past tense for this, but it should be consistent.